lib/notes.js

```diff
@@ -3,11 +3,17 @@
 const { default: HackMD } = require('@hackmd/api')
 const defaultNotesTemplate = require('./default-notes-template')
 
-module.exports.create = function createNote (notesTemplate, opts) {
+module.exports.create = async function createNote (notesTemplate, opts) {
   const note = typeof notesTemplate === 'function' ? notesTemplate(opts) : notesTemplate || defaultNotesTemplate(opts)
-  const hackmd = new HackMD()
 
-  return hackmd.newNote(note)
+  try {
+    const hackmd = new HackMD()
+    return await hackmd.newNote(note)
+  } catch (e) {
+    console.error('failed to create hackMD note:', e.message)
+    console.log(`note would have been:\n${note}`)
+    return null
+  }
```

Comment:
This is unrelated, but I am guessing it was failing for your testing and this helped make the message more clear? Might just be nice to do in a separate commit.

Reply:
it is a separate commit... just in the same PR  😄

Reply:
oh sorry, I probably didn't scroll back up to see that. Ignore me.